### DISCUSSION TITLE
User Profile modification

### DISCUSSION THREAD

**Discussion by lavabyrd** [soundshed/soundshed-app#14] [Q&A] [answered]
Just managed to create my profile and use a stacktrace as the password instead of a LastPass generated password (my fault, not the app). But I don't see a way to modify my profile, I can "sign out" but not reset or change anything. Doesn't look like the member function is exposed on soundshed.com yet either.

Also spotted an issue around the login window and it being stuck in the "create new member" field with no way to go back. I'll get an issue open for it.

**Comment by soundshed** [comment, marked as answer]
Thanks, yes that side of it's not really properly filled out yet. As an aside I also need to figure out if we're going to let people login to their PG ToneCloud account or not, that would unlock things like seeing your existing favourites and possibly downloads from that tonecloud etc, but I can see it getting confusing that there are two accounts (soundshed and PG).
